index.js

```diff
@@ -2,6 +2,9 @@
  * Module dependencies
  */
 var assert = require('assert')
+var get = require('get-value')
+var set = require('set-value')
+var isObject = require('is-kindof').isObject;
```

Comment:
I'm not too fond of introducing this as a dependency. Don't see much added value over `assert.equal(typeof val, 'object')`.